controls/flex_exists.rb

```diff
@@ -2,21 +2,22 @@
 
 plan_origin = ENV['HAB_ORIGIN']
 plan_name = input('plan_name', value: 'flex')
- 
+
 control 'core-plans-flex-exists' do
   impact 1.0
   title 'Ensure flex exists'
   desc '
-  Verify flex by ensuring /bin/flex exists'
+  Verify flex by ensuring bin/flex exists'
   
   plan_installation_directory = command("hab pkg path #{plan_origin}/#{plan_name}")
   describe plan_installation_directory do
     its('exit_status') { should eq 0 }
     its('stdout') { should_not be_empty }
+    its('stderr') { should be_empty }
   end
 
-  command_relative_path = input('command_relative_path', value: '/bin/flex')
-  command_full_path = plan_installation_directory.stdout.strip + "#{command_relative_path}"
+  command_relative_path = input('command_relative_path', value: 'bin/flex')
+  command_full_path = File.join(plan_installation_directory.stdout.strip, command_relative_path)
   describe file(command_full_path) do
     it { should exist }
```

Comment:
Can you add a check for flex being an executable?

Reply:
Done.  Cheers!